src/rouge/worker/worker.py

```
@@ -113,42 +114,131 @@ def _handle_shutdown(self, signum, frame):
+            .order("created_at", desc=True)
+            .limit(1)
+            .maybe_single()
+            .execute()
+        )
+
+        if response.data is None or response.data.get("adw_id") is None:
+            raise ValueError(f"No workflow_completed comment found for issue {issue_id}")
+
+        return response.data["adw_id"]
+
+    def _execute_patch_workflow(self, issue_id: int, description: str) -> tuple[str, bool]:
```

Ruff (0.14.11)

Unused method argument: `description`

(ARG002)